pkg/gardenadm/botanist/botanist.go

```diff
@@ -192,7 +191,6 @@ func newSeedObject(ctx context.Context, shootObj *shootpkg.Shoot) (*seedpkg.Seed
 
 func newShootObject(ctx context.Context, projectName string, cloudProfile *gardencorev1beta1.CloudProfile, shoot *gardencorev1beta1.Shoot) (*shootpkg.Shoot, error) {
 	shoot.Status.TechnicalID = gardenerutils.ComputeTechnicalID(projectName, shoot)
-	shoot.Status.Gardener = gardencorev1beta1.Gardener{Name: "gardenadm", Version: version.Get().GitVersion}
```

Comment:
I think it's pretty likely that we anyway need this later when we instruct a `gardenlet` (or something similar) to reconcile such an autonomous shoot. Should we just leave it in?